docs/book/src/reference/compiler_intrinsics.md

```diff
@@ -372,3 +372,18 @@ Runtime bound checks are not generated, and must be done manually when and where
 
 - `item` is a reference to an array or a reference to a slice;
 - when `index` is a literal, it must be smaller than `item` length;
+
+---
+
+```sway
+__dbg(value: T) -> T
+```
+
+**Description:** Automatically calls the `Debug` trait on the passed `value`, with file, line and column information. The value is returned to allow be used inside any expression.
+
+Only generate the expected output in `Debug` mode, when in `Release` behaves as identity function, just returning the passed value.
+To enable the output generation in "Release", must be enabled in the project `Forc.toml`
```

Comment:
This formulation sounds confusing to me. What exactly must be enabled and how? Having an example here how to enable it in the `Forc.toml` would IMO be very beneficial.

```suggestion
To enable the output generation in `Release` mode, must be enabled in the project's `Forc.toml`.
```

Reply:
Done.